src/main/java/org/opentestsystem/delivery/logging/EventParser.java

```diff
@@ -24,7 +25,7 @@
    * @return
    */
   public Optional<EventInfo> parsePreHandle(final HttpServletRequest request, final Object handler, final EventLogger logger) {
-    return Optional.of(EventInfo.create(request.getPathInfo(), getEventDataFields(request)));
+    return Optional.of(EventInfo.create(getPath(request), getEventDataFields(request)));
   }
 
```

Comment:
this is a fine solution, but I wonder if it'd be safer to instead allow EventInfo.create() to accept a null event.